# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditorForwards.h
# Commit: a43e95a534eb
# Full Hash: a43e95a534eb5c13883cc2a61fb64c15fe29e674
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-10-05 03:40:21
# Regressor Bug: 1792654
# File Overlap Count: 1
# Description:
#   Bug 1792654 - Make `CreateNodeResultBase`, `MoveNodeResult`, `SplitNodeResult` and `SplitRangeOffResult` handle caret point with a common base class r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D158335
# ==============================================================================

diff -r 18196c5fab62 -r a43e95a534eb editor/libeditor/EditorForwards.h
--- a/editor/libeditor/EditorForwards.h	Tue Oct 04 21:13:53 2022 +0000
+++ b/editor/libeditor/EditorForwards.h	Tue Oct 04 21:36:57 2022 +0000
@@ -76,6 +76,7 @@
 
 class AutoPendingStyleCacheArray;  // mozilla/PendingStyles.h
 class AutoSelectionRangeArray;     // mozilla/EditorUtils.h
+class CaretPoint;                  // mozilla/EditorUtils.h
 class ChangeStyleTransaction;      // mozilla/ChangeStyleTransaction.h
 class CSSEditUtils;                // mozilla/CSSEditUtils.h
 class EditActionResult;            // mozilla/EditorUtils.h